x-pack/elastic-agent/pkg/agent/cmd/enroll.go

```diff
@@ -187,6 +187,11 @@ func enroll(streams *cli.IOStreams, cmd *cobra.Command, flags *globalFlags, args
 		}
 	}
 
+	// enroll is invoked either manually or from install with redirected IO
+	// no need to log to file
+	cfg.Settings.LoggingConfig.ToFiles = false
+	cfg.Settings.LoggingConfig.ToStderr = true
```

Comment:
Nice! Does this need to be set on any other subcommands?

Reply:
no we use just `install`, `uninstall`, `upgrade` and `reexec (on win)` where we don't use logger
`enroll` where we use it
`watcher` has its own configuration
and run which is a normal state

Reply:
Thanks for the response.